uxpin-merge/MUI-Merge
# Upgrade merge cli

## Overview
Upgrade the merge cli package to the latest version: `3.0.3`. 
Check out the latest changes here: https://github.com/UXPin/uxpin-merge-tools/blob/master/packages/uxpin-merge-cli/CHANGELOG.md

## How to test
* Pull this branch in the repo
* `yarn install --force`
* check common merge-cli commands:
   * `npx uxpin-merge push -t`
   * `npx uxpin-merge push -t -tag`
   * `npx uxpin-merge push -t -branch`
   
More info on the commands can be found here: https://www.uxpin.com/docs/merge/cli-tool/
